hack/generate-manifest.sh

```diff
@@ -106,6 +108,11 @@ case $key in
     PROXY=false
     shift
     ;;
+    --endpointslice)
+    PROXY=true
```

Comment:
@antoninbas `AntreaProxy` and `EndpointSlice` can be enabled at the same time. However it's unnecessary to enable both arguments. We can see here if `EndpointSlice` is enabled, `AntreaProxy` will also be set as enabled.

Reply:
Sorry, is this the answer to one of my questions? I'm confused

Reply:
>One of my comments has not been addressed: I do not see any mention anywhere of the behavior if 1) AntreaProxy and EndpointSlice feature gates are enabled; 2) the EndpointSlice API is disabled in K8s. IMO, the behavior should at least be mentioned in docs/feature-gates.md and should probably also be included in the commit message.

This is the answer of question 1).

Reply:
I am asking about the behavior when EndpointSlice is enabled in Antrea but the EndpointSlice API is disabled / not supported in Kubernetes. Is there a crash? Is there an error log message? Is it a no-op?

Reply:
The restriction of using `EndpointSlice` in Antrea should be explained in docs. The explain is that this feature can be set as enabled if only that Kubernetes 1.16 (alpha) or later with EndpointSlice enabled. For users of Antrea, the behavior when EndpointSlice is enabled in Antrea but the EndpointSlice API is disabled / not supported in Kubernetes should be avoid. So far, There is not a exact evaluation about the unwanted behavior.

Reply:
> So far, there is not an exact evaluation about the unwanted behavior.
Could we make a quick test to have a clear idea on that?

Reply:
I've tested this scenario on a single node minikube cluster with Kubernetes v1.16.1.
The agent was not crash and no services/endpoints could be watched, and thus the agent can not connect to the controller.
There are logs in the agent indicate that the EndpointSlice resource is unavailable. 
```
E0202 04:53:35.852020       1 reflector.go:178] pkg/mod/github.com/tnqn/client-go@v0.18.4-1/tools/cache/reflector.go:125: Failed to list *v1beta1.EndpointSlice: the server could not find the requested resource (get endpointslices.discovery.k8s.io)
```
 
I think we can first try to check if the EndpointSlice resource exists before really starting Antrea Proxy with EndpointSlice enabled.